api/config/cdr_config_perf.json

```diff
@@ -7,6 +7,14 @@
       "servicePerimeter": "accessPolicies/228353087260/servicePerimeters/terra_perf_aou_perf",
       "authDomainName": "all-of-us-registered-perf",
       "authDomainGroupEmail": "all-of-us-registered-perf@perf.test.firecloud.org"
+    },
+    {
+      "accessTierId": 2,
+      "shortName": "controlled",
+      "displayName": "Fake Controlled Tier For Perf",
+      "servicePerimeter": "accessPolicies/228353087260/servicePerimeters/terra_perf_aou_perf_2",
+      "authDomainName": "all-of-us-controlled-perf",
+      "authDomainGroupEmail": "all-of-us-controlled-perf@perf.test.firecloud.org"
```

Comment:
The process for authDomain config creation is a little goofy: populate the `authDomainName` here, run a `./project.rb create-auth-domain` which uses this value, and then populate `authDomainGroupEmail` based on what was returned.

What's even better is that this `authDomainGroupEmail` is not used.  (the `:auth_domain_group_email` below is used).   We could remove it but I'd prefer to wait for [RW-6426](https://precisionmedicineinitiative.atlassian.net/browse/RW-6426) to handle it holistically.